monitor/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:18.04
```

Comment:
is there a way to use smaller base image or use an image we prepared which have all pre required package installed?  It takes some time to install all required packaged after pull the base image.